lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
       margin: 0;
       padding: 0;
       list-style: none;
 
       .courses-listing-item {
-        width: flex-grid(4);
-        @include margin-right(flex-gutter());
-        @include float(left);
+        height: 360px;
+
+        @include media($mobile) {
```

Comment:
I'd prefer that we think of our rules as mobile-first (defining mobile as the base rules and then extending/overriding things like layout for larger breakpoints). In this case, that means moving the columns and margins-based rules to the base definition, like so: 

````scss
.courses-listing-item {
    @include margin-right(flex-gutter());
    @include float(left);
    height: 360px;
    @include span-columns(4);
    margin: 15px 0px;

    @include media($tablet) {
        ...
    }

    @include media($desktop) {
        ...
    }

    @include media($xl-desktop) {
        ...
    }
}
````